test/fixtures/rules/std_rule_cases/RF06.yml

```diff
@@ -120,7 +132,7 @@ test_pass_column_reference_prefer_quoted_tsql:
 
 test_fail_column_reference_prefer_quoted_ansi:
   fail_str: |
-    SELECT 123 AS foo;
+    SELECT foo;
```

Comment:
This test doesn't appear to be doing what it should be any more

Reply:
Or rather, cna we keep this as an alias?

Reply:
Having picked through this PR and the others linked to it - I'm not so sure. I think what's happened is that RF06 has got really confusing. I'm going to write up something more comprehensive.

It doesn't help that the docs for this rule are fairly unhelpful.

Reply:
Hi @WittierDinosaur , this test `test_fail_column_reference_prefer_quoted_ansi` looks correct to me. With respect to the configurations in the test: `prefer_quoted_identifiers: true` the query: 
```
SELECT foo;
```
should fail as the correct query under this configuration would be 
```
SELECT "foo";
```

What do you think?